envi/__init__.py

```diff
@@ -753,6 +753,9 @@ def getNumStackArgs(self, emu, argc):
         rargs = [ v for (t, v) in self.arg_def if t == CC_REG ]
         return max(argc - len(rargs), 0)
 
+    def getStackArgOffset(self, emu, argc):
```

Comment:
add function docstring like, 'Returns offset to first stack argument'

add unittest for this new api to:
https://github.com/vivisect/vivisect/blob/master/vdb/testmods/callingconventions.py